packages/app/src/cli/services/webhook/send-app-uninstalled-webhook.ts

```diff
@@ -31,6 +31,7 @@ export async function sendUninstallWebhookToAppServer(
 
   options.stdout.write('Sending APP_UNINSTALLED webhook to app server')
 
+  await sleep(3)
```

Comment:
Did you mean to leave this in?

Reply:
Yes, the webhook was firing at the same time as the actual process started, which led to it failing most of the times. This is so we back off for a few seconds to give the app time to spin up.